hs_model_program/forms.py

```diff
@@ -12,18 +12,47 @@
 class mp_form_helper(BaseFormHelper):
     def __init__(self, allow_edit=True, res_short_id=None, element_id=None, element_name=None, *args, **kwargs):
 
+        files = kwargs.pop('files')
+        file_data = {}
+        for f in files.all():
+            short_url = f.resource_file.name
+            name = short_url.split('/')[-1]
+            file_data[name] = short_url
+
+        options = '\n'.join(['<option value=%s>%s</option>'%(value, key) for key, value in file_data.iteritems() ])
```

Comment:
There is no escaping of these strings, which why generating HTML outside of templates is something you need to be so careful about.